src/content/en/fundamentals/discovery-and-monetization/payment-request/deep-dive-into-payment-request.md

```diff
@@ -52,9 +52,12 @@ two major Chrome versions.
 
 ## Feature Detect
 
-Before we start covering how to use PaymentRequest, we should feature detect to
-ensure it's available and fallback to a traditional checkout page in browsers
-that don't support it. The feature detect is simply:
+PaymentRequest has been available since Chrome 53 for Android and has been
```

Comment:
Do one of the following as appropriate for all occurrences of `PaymentRequest`:

- When referring to the API in general say, 'Payment Request API'.
- When referring to an object say, `PaymentRequest` object (or instance, as appropriate). Notice where the backticks are.
- When referring generically to a payment request say, 'payment request'. 

Please do this throughout your article.